session_4/blogspot/app/views/articles/edit.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1 class="text-center">Editing Article</h1>
+<br/>
+<%= render 'form', article: @article %>
+<br/>
+<%= link_to 'Show', @article, class:"btn btn-outline-primary" %> |
+<%= link_to 'Back', articles_path, class:"btn btn-outline-primary" %>
```

Comment:
Use different colors for show and back buttons - maybe primary and info?

Reply:
Ohk.. I'll change that